docs/exts/provider_yaml_utils.py

```diff
@@ -68,7 +68,7 @@ def load_package_data(include_suspended: bool = False) -> list[dict[str, Any]]:
         try:
             jsonschema.validate(provider, schema=schema)
         except jsonschema.ValidationError:
-            raise Exception(f"Unable to parse: {provider_yaml_path}.")
+            raise ValueError(f"Unable to parse: {provider_yaml_path}.")
```

Comment:
Sys.exit  and message containing error message from validation explaining which faile failed validation

Reply:
Wondering, do we need to catch the exception here if we would like to raise an error with content of it?